scripts/pre.sh

```diff
@@ -45,17 +44,17 @@ elif [ "$TARGET_DIR" == "MeshLib/local" ]; then
     URL="127.0.0.1:8000/$TARGET_DIR"
 fi
 
-
-mkdir -p ${TARGET_DIR}/html
-rm -rf ${TARGET_DIR}/html/*
-if [ -f ../MeshLib/scripts/doxygen/generate_doxygen_layout.sh ]; then
-    MODULES=(Py Cpp Main)
-    for MODULE in ${MODULES[*]}
-    do
-        ../MeshLib/scripts/doxygen/generate_doxygen_layout.sh $MODULE $URL
-        # force Doxygen to use the custom HTML header
-        sed -e "s|HTML_HEADER\s*=.*|HTML_HEADER = html_header.html|" -i.bak Doxyfile${MODULE}
-        # force Doxygen to use the custom output directory
-        sed -e "s|OUTPUT_DIRECTORY\s*=.*|OUTPUT_DIRECTORY = ${TARGET_DIR}|" -i Doxyfile${MODULE}
-    done
+if [ ! -f ../MeshLib/scripts/doxygen/generate_doxygen_layout.sh ]; then
+    echo "[ERROR] Can not found script to generate doxygen layout files"
+    exit 1
 fi
+
+MODULES=(Py Cpp Main)
+for MODULE in ${MODULES[*]}
+do
+    ../MeshLib/scripts/doxygen/generate_doxygen_layout.sh $MODULE $URL
+    # force Doxygen to use the custom HTML header
+    sed -e "s|HTML_HEADER\s*=.*|HTML_HEADER = html_header.html|" -i.bak Doxyfile${MODULE}
+    # force Doxygen to use the custom output directory
+    sed -e "s|OUTPUT_DIRECTORY\s*=.*|OUTPUT_DIRECTORY = ${TARGET_DIR}|" -i Doxyfile${MODULE}
```

Comment:
do we change output dir for Main module?

Reply:
Yes. It can also be "MeshLib/dev" or "MeshLib/local"